src/statsd.ts

```diff
@@ -11,12 +11,14 @@ export interface StatsDOptions {
     statsdClient?: StatsdClient;
 }
 
+const DEBUG_STATSD: boolean = false;
```

Comment:
this looks like it was left over from testing, i don't think we need this anymore.